<launch>

    <include file="$(find realsense2_camera)/launch/rs_camera.launch" />   
     
    <!-- Launch the human_location_processor node -->    
    <node name="human_location_processor" pkg="robot_ctrl" type="human_location_processor.py" output="screen">
        <!-- Add necessary parameters for the human_location_processor node -->
        <param name="confidence_threshold" value="0.75" />
        <param name="iou_threshold" value="0.45" />
        <param name="agnostic_nms" value="true" />
        <param name="maximum_detections" value="1000" />
        <param name="weights" value="$(find yolov5_ros)/src/yolov5/weights/yolov5s.pt" />
        <param name="device" value="cpu" />
        <param name="inference_size_w" value="640" />
        <param name="inference_size_h" value="480" />
        <param name="half" value="false" />
    </node>


</launch>

